app/presenters/organisations/show_presenter.rb

```diff
@@ -127,6 +134,10 @@ def has_definite_article?(phrase)
       phrase.downcase.strip[0..2] == "the"
     end
 
+    def english_only_subscription?
```

Comment:
All email subscriptions are english only, so I think this flag might be better renamed to custom_signup_button_test or similar?

Reply:
Changed